crates/history/src/any.rs

```diff
@@ -0,0 +1,203 @@
+use std::borrow::Cow;
+
+#[cfg(feature = "serde")]
+use serde::de::DeserializeOwned;
+#[cfg(feature = "serde")]
+use serde::Serialize;
```

Comment:
There's no `serde` feature now.

Reply:
When an optional dependency is added, a feature with the same name is implicitly registered.

> Optional dependencies implicitly define a feature of the same name as the dependency. This means that the same cfg(feature = "gif") syntax can be used in the code, and the dependency can be enabled just like a feature such as --features gif (see Command-line feature options below).

https://doc.rust-lang.org/cargo/reference/features.html

Reply:
TIL